test/sync_fixtures.hpp

```diff
@@ -301,10 +301,10 @@ class HTTPRequestClient {
 
     util::PrefixLogger logger;
 
-    HTTPRequestClient(util::Logger& logger, const util::network::Endpoint& endpoint, const util::HTTPRequest& request)
-        : logger{"HTTP client: ", logger}
+    HTTPRequestClient(util::Logger& l, const util::network::Endpoint& endpoint, const util::HTTPRequest& request)
```

Comment:
Can we not name things `l`? Why did this need to change?

Reply:
Sorry